### PR TITLE
New request parameters enrich

### PR DESCRIPTION
**Pull request by prakhar-j**
- Added support for verifiedPhysical, expandedInterests, maxMaids and Epsilon in PersonResponse
[ch134138]

### CODE REVIEW

File: java-common-artifacts/src/main/java/com/fullcontact/apilib/models/Request/PersonRequest.java [hunk 1]
```diff
@@ -17,4 +17,6 @@
   private Confidence confidence;
   @Builder.Default private boolean infer = true;
   private List<String> dataFilter;
+  private boolean verifiedPhysical, expandedInterests;
+  private Integer maxMaids;
```
Comment (kerdosa) [inline java-common-artifacts/src/main/java/com/fullcontact/apilib/models/Request/PersonRequest.java:21]: Please add javadoc documentation what these fields for.

File: java-common-artifacts/src/main/java/com/fullcontact/apilib/models/Response/PersonResponse.java [hunk 2]
```diff
@@ -25,6 +26,8 @@
       avatar,
       website;
   private Details details;
+  private Boolean verifiedPhysical;
+  private Map<String, String> epsilon;
```
Comment (kerdosa) [inline java-common-artifacts/src/main/java/com/fullcontact/apilib/models/Response/PersonResponse.java:31]: Add javadoc documentation about new fields.

**Review by kerdosa — APPROVED** [review]
Add some documentation.

File: java-common-artifacts/src/main/java/com/fullcontact/apilib/models/Request/PersonRequest.java (hunk 1 above)
Comment (prakhar-j) [reply java-common-artifacts/src/main/java/com/fullcontact/apilib/models/Request/PersonRequest.java:25]: Done

File: java-common-artifacts/src/main/java/com/fullcontact/apilib/models/Response/PersonResponse.java (hunk 2 above)
Comment (prakhar-j) [reply java-common-artifacts/src/main/java/com/fullcontact/apilib/models/Response/PersonResponse.java:33]: Done

**Pull request merged**